Formula/pplacer.rb

```diff
@@ -0,0 +1,34 @@
+class Pplacer < Formula
+  # Matsen_2010: "https://dx.doi.org/10.1186/1471-2105-11-538"
```

Comment:
```suggestion
  # cite Matsen_2010: "https://doi.org/10.1186/1471-2105-11-538"
```
See the output of `brew cite --ruby 10.1186/1471-2105-11-538` from `brew tap brewsci/base`